content/open-source/melange/faq.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "melange FAQs"
+type: "article"
+lead: "Frequently asked questions about melange"
+description: "Frequently asked questions about melange"
+date: 2022-10-17T11:07:52+02:00
+lastmod: 2022-10-17T11:07:52+02:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "melange"
+weight: 900
+toc: true
+---
+
+### How are melange packages defined?
+melange apks are defined declaratively using a YAML file.
+
+### Are melange apks compatible with Alpine?
+Yes, melange apks are built to be compatible with apk-based systems including Alpine.
+
+### Can I mix Alpine and Wolfi package repositories to create my melange build environment?
+No, it's not possible to mix Alpine apks with Wolfi apks. If you have unmet dependencies, you'll need to build those first as separate packages.
+
+### Is it mandatory to sign packages with a melange key?
+Signing packages is not mandatory, but it is a recommended practice, because it allows users and automated systems to verify that the package they downloaded was built by the same person who signed it, and that it hasn't been tampered with.
```

Comment:
I think it's worthwhile to call out that it can cause problems. I could see this and the next question being combined, alternatively

```suggestion
Signing packages is not mandatory, but it is a recommended practice that is enforced by some systems. Signing packages allows users and automated systems to verify that the package they downloaded was built by the same person who signed it, and that it hasn't been tampered with.
```

Reply:
I think it makes a lot of sense to combine them!